packages/mobile/package.json

```diff
@@ -15,25 +15,27 @@
     "nativewind": "^4.0.1",
     "react": "18.2.0",
     "react-native": "0.74.5",
-    "react-native-gesture-handler": "^2.18.1",
-    "react-native-reanimated": "^3.15.0",
+    "react-native-gesture-handler": "~2.16.1",
+    "react-native-reanimated": "~3.10.1",
     "react-native-remix-icon": "^0.4.0",
-    "react-native-svg": "^15.6.0",
+    "react-native-svg": "15.2.0",
     "tailwindcss": "^3.4.10"
   },
   "devDependencies": {
     "@babel/core": "^7.20.0",
-    "@react-native-async-storage/async-storage": "^2.0.0",
-    "@react-native-community/datetimepicker": "^8.2.0",
+    "@react-native-async-storage/async-storage": "1.23.1",
+    "@react-native-community/datetimepicker": "8.0.1",
     "@react-native-community/slider": "^4.5.2",
     "@storybook/addon-ondevice-actions": "^7.6.20",
     "@storybook/addon-ondevice-controls": "^7.6.20",
     "@storybook/react-native": "^7.6.20",
-    "@types/react": "~18.2.45",
+    "@types/react": "^18.3.4",
+    "@types/react-native": "^0.73.0",
     "babel-loader": "^8.3.0",
+    "clsx": "^2.1.1",
     "react-dom": "18.2.0",
-    "react-native-safe-area-context": "^4.10.9",
-    "typescript": "^5.1.3"
+    "react-native-safe-area-context": "4.10.5",
+    "typescript": "~5.3.3"
```

Comment:
Any reason for downgrading these packages @amireds ?

Reply:
if you run expo doctor, it tells you the dependency version required to run this current react native version optimally. What I did was to fix the dependency version

`npx expo doctor --fix-dependencies`
`npx expo doctor`